octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/util/error/ErrorComposite.java

```diff
@@ -0,0 +1,180 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.hpe.octane.ideplugins.eclipse.ui.util.error;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang.StringUtils;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.layout.FillLayout;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Label;
+import com.hpe.adm.nga.sdk.exception.OctaneException;
+import com.hpe.adm.nga.sdk.model.ErrorModel;
+import com.hpe.adm.nga.sdk.model.FieldModel;
+import com.hpe.adm.octane.ideplugins.services.util.Util;
+import com.hpe.octane.ideplugins.eclipse.ui.util.TruncatingStyledText;
+import com.hpe.octane.ideplugins.eclipse.ui.util.resource.SWTResourceManager;
+
+public class ErrorComposite extends Composite {
+    
+    public static final String ERORR_MODEL_FIELD_DESCRIPTION_TRANSLATED = "description_translated";
+    public static final String ERORR_MODEL_FIELD_DESCRIPTION = "description";
+    public static final String ERORR_MODEL_FIELD_PROPERTIES = "properties";
+    public static final String ERORR_MODEL_FIELD_ERROR_CODE = "error_code";
+    public static final String ERORR_MODEL_FIELD_HTTP_STATUS_CODE = "httpStatusCode";
+    public static final String ERORR_MODEL_FIELD_CORRELATION_ID = "correlation_id";
+    
+    private Composite compositeExceptionData;
+    private Composite compositeBtns;
+    private Label btnSeparator;
+
+    /**
+     * Create the composite.
+     * @param parent
+     * @param style
+     */
+    public ErrorComposite(Composite parent, int style) {
+        super(parent, style);
+        setLayout(new GridLayout(1, false));
+        
+        setBackground(SWTResourceManager.getColor(SWT.COLOR_LIST_BACKGROUND));
+        setForeground(SWTResourceManager.getColor(SWT.COLOR_LIST_FOREGROUND));
+        setBackgroundMode(SWT.INHERIT_FORCE);
+        
+        compositeExceptionData = new Composite(this, SWT.NONE);
+        compositeExceptionData.setLayout(new GridLayout(1, false));
+        compositeExceptionData.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, false, 1, 1)); 
+        
+        btnSeparator = new Label(this, SWT.SEPARATOR | SWT.HORIZONTAL);

```

Comment:
I think this is not the best name for this separator, it's a bit confusing at first sight. Maybe something like lblSeparator?

Reply:
ops